### PR TITLE
fixes #82 #88 and sponsorship update, misc

### LINKED ISSUE #82: Cleanup of old, unused files and directory structure

### PR DESCRIPTION
**Pull request by nicklambourne**
- #82 Cleanup of old unused files & directory structure (mostly moving images and uploads out of root directory)
- #88 Have added limited social media meta and generic image
- Updated hardcoded year (footer and licence)
- Updated sponsor images to those provided by aginicX and Softwire

### CODE REVIEW

**Comment by nicklambourne** [comment]
Sorry this touches so much in a single PR :(

**Comment by mitchmcdee** [comment]
damn nice! its a big hard to tell what these changes are though-- can you provide some rough screenshots? :^)

**Comment by nicklambourne** [comment]
screen shot 2018-03-08 at 2 05 30 am
screen shot 2018-03-08 at 2 05 51 am

**Comment by nicklambourne** [comment]
screen shot 2018-03-08 at 2 08 12 am
screen shot 2018-03-08 at 2 09 56 am

File: _includes/foot.html [hunk 1]
```diff
@@ -9,17 +9,17 @@ <h5 class="white-text">Come, chat with us!</h5>
       <div class="col l4 offset-l2 s12">
         <h5 class="white-text">Links</h5>
         <ul>
-          <li><a class="grey-text text-lighten-3" href="https://www.facebook.com/uqcomputing" target="_blank"><img height="14" width="14" src="{{site.baseurl}}/img/facebk.svg"></img> Facebook</a></li>
-          <li><a class="grey-text text-lighten-3" href="https://github.com/UQComputingSociety/" target="_blank"><img height="14" width="14" src="{{site.baseurl}}/img/github.svg"></img> GitHub</a></li>
-          <li><a class="grey-text text-lighten-3" href="http://twitter.com/UQComputing" target="_blank"><img height="14" width="14" src="{{site.baseurl}}/img/twtr.svg"></img> Twitter</a></li>
-          <li><a class="grey-text text-lighten-3" href="https://slack.uqcs.org.au/" target="_blank"><img height="14" width="14" src="{{site.baseurl}}/img/slack.svg"></img> Slack</a></li>
+          <li><a class="grey-text text-lighten-3" href="https://www.facebook.com/uqcomputing" target="_blank"><img height="14" width="14" src="{{site.baseurl}}/img/social/facebk.svg"></img> Facebook</a></li>
```
Comment (draganmarjanovic) [inline _includes/foot.html]: Minor but I'm not a fan of these abbreviations `facbk` and `twtr`. Because if you choose to reuse it somewhere else you probably can't just remember the basic path and then replace with facebook/twitter/etc but instead would have to check the filename.

File: css/uqcs.css [hunk 2]
```diff
@@ -89,7 +89,7 @@ body {
 
 .navbar-fixed
 {
-    margin-bottom: 64px; /* Default height of Materialize Navbar */
+    margin-bottom: 56px; /* Default height of Materialize Navbar */
```
Comment (draganmarjanovic) [inline css/uqcs.css]: What's the motivation behind this change? Before/after pics?

Comment (nicklambourne) [reply css/uqcs.css]: There was a gap here that only showed up on mobile/smallest view size.
screen shot 2018-03-12 at 10 32 04 am
screen shot 2018-03-12 at 10 32 36 am

File: _includes/foot.html (hunk 1 above)
Comment (nicklambourne) [reply _includes/foot.html]: Agreed. I have changed these to the full names :)

File: css/uqcs.css (hunk 2 above)
Comment (draganmarjanovic) [reply css/uqcs.css]: Ah yep, good catch!

**Comment by draganmarjanovic** [comment]
@nicklambourne Good job, but plz, smaller PR's ❤️
